src/client.js

```diff
@@ -35,12 +35,15 @@ function initSocket() {
 global.socket = initSocket();
 
 const component = (
-  <Provider store={store} key="provider">
-    <ReduxRouter routes={getRoutes(store)} />
-  </Provider>
+  <ReduxRouter routes={getRoutes(store)} />
 );
 
-ReactDOM.render(component, dest);
+ReactDOM.render(
+  <Provider store={store} key="provider">
```

Comment:
No idea. No good reason, probably. Must've been part of an array of components at some point.